.circleci/docker-php-circleci.ini

```diff
@@ -0,0 +1,29 @@
+;;;;;;;;;;;;;;;;;;;
+; Resource Limits ;
+;;;;;;;;;;;;;;;;;;;
+
+; Maximum execution time of each script, in seconds
+; http://php.net/max-execution-time
+; Note: This directive is hardcoded to 0 for the CLI SAPI
+max_execution_time = 60
```

Comment:
Is this required if the CLI is hardcoded?

Reply:
Good point, let's get rid of it